src/pages/CreatePool/CreatePoolDialog.tsx

```diff
@@ -101,8 +101,20 @@ export default function ReviewCreatePool({
       }
       resetFields()
     } catch (err) {
-      console.error(err)
-      enqueueToast("error", "Unable to deploy Permissionless Pool")
+      console.error({ err })
+      const er = err as { error: { data: { message: string } } }
+      if (
+        er.error.data.message.includes(
+          "SafeERC20: approve from non-zero to non-zero allowance",
```

Comment:
nit: If we're looking for a substring then `.includes` is good, but if this is the entire message, can we use some kind of string equality check instead?

Reply:
the full message is longer. if you find a unique way of parsing lmk